releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,576 @@
+
+    Added a configuration item `max-error`. The default value is 0. When the value is greater than 0, the max-error function is enabled. If an error occurs in a row during encoding, a record containing this row is added to `type_error_v1` and this row is ignored. When rows with errors exceed the threshold, TiDB Lightning exits immediately.
+
+    Matching the `max-error` configuration, the `lightning_task_info` configuration item records the name of the database that reports a data saving error.
+
+    This feature does not cover all types of errors, for example, syntax errors are not applicable.
+
+    [User document](/tidb-lightning/tidb-lightning-error-resolution.md#type-error)
+
+### Data sharing and subscription
+
+- Support replicating 100K tables simultaneously
+
+    By optimizing the data processing flow, TiCDC reduces the resource consumption of processing incremental data for each table, which greatly improves the replication stability and efficiency when replicating data in large clusters. The result of an internal test shows that TiCDC can stably support replicating 100,000 tables simultaneously.
+
+    [User document]()
+
+### Deployment and maintenance
+
+- Enable new collation rules by default
+
+    TiDB has supported new collation rules since v4.0, which behave the same as MySQL in the case-insensitive, accent-insensitive, and padding rules. The new collation rules are controlled by the `new_collations_enabled_on_first_bootstrap` parameter, which was disabled by default. Since v6.0, TiDB enables new collation rules by default. Note that this configuration takes effect only for the TiDB clusters first initialized.
+
+    [User documentation](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap)
+
+- ​​Accelerate leader balancing after restarting TiKV nodes
+
+    After a restart, TiKV nodes need to redistribute the unevenly scattered leaders for load balance. In large-scale clusters, leader balancing time is positively correlated with the number of Regions. For example, the leader balancing of 100K Regions can take 20-30 minutes, which is prone to performance issues and stability risks due to uneven load. TiDB 6.0 provides a parameter to control the balancing concurrency and adjusts the default value to 4 times of the original, which greatly shortens the leader rebalancing time and accelerates the business recovery after a restart of TiKV nodes.
+
+    [User documentation](/pd-control.md#scheduler-show--add--remove--pause--resume--config)
+
+- Support canceling the automatic update of statistics
+
+    Statistics are one of the most important basic data that affects SQL performance. To ensure the completeness and timeliness of statistics, TiDB updates statistics periodically. Since v6.0, you can cancel the automatic update of statistics manually, which avoids any SQL performance impact caused by resource contention of the automatic update task.
+
+    [user documentation](/statistics.md#automatic-update)
+
```

Comment:
```suggestion

### TiDB Enterprise Manager

TiDB Enterprise Manager (TiEM) is an enterprise-level database management platform based on the TiDB database, which aims to help users deploy, monitor, back up, and scale TiDB clusters in on-premises or public cloud environments.

TiEM not only provides full lifecycle visual management for TiDB clusters, but also provides one-stop parameter management, version upgrades, cluster clone, active-standby cluster switching, data import and export, data replication, data backup and restore services. TiEM can improve the efficiency of DevOps on TiDB and reduce the DevOps cost for enterprises.

Currently, TiEM is for the [TiDB Enterprise](https://en.pingcap.com/tidb-enterprise/) edition only. To get TiEM and its documentation, contact to us via the [TiDB Enterprise](https://en.pingcap.com/tidb-enterprise/) page.

```

Reply:
Suggestions for the second and third parts:

TiEM not only provides full lifecycle visual management for TiDB clusters but also provides one-stop services: parameter management, version upgrades, cluster clone, active-standby cluster switching, data import and export, data replication,  and data backup and restore services. With TiEM, you can improve the efficiency of DevOps on TiDB and reduce the DevOps cost for enterprises.

Currently, TiEM is for the [TiDB Enterprise](https://en.pingcap.com/tidb-enterprise/) edition only. To get TiEM and its documentation, contact us via the [TiDB Enterprise](https://en.pingcap.com/tidb-enterprise/) page.